docs/animations/AnimationSet.md

```diff
@@ -71,6 +69,7 @@ You can change the way how the animation interpolates between keyframes by defin
 - AnimationSet has endless possibility. Here is an example of creating popup effect
 
     **Sample Code**
+
     ```csharp
```

Comment:
Is this is a fix for MarkdownTextBlock issue or is this a general improvement?

Reply:
Both, that is proper Markdown. Markdown requires a new line to be two returns long.

Reply:
But displaying property without this fix!

Reply:
It isn't proper, and it doesn't display properly in the Sample App.

Reply:
Displayed properly in old sample app